client/src/containers/backend/Form/UserClaims.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component, PropTypes } from 'react';
+import { Form, Text } from 'components/backend';
+import { makersAPI, usersAPI } from 'api';
+import { connect } from 'react-redux';
+import { entityStoreActions } from 'actions';
+import { entityUtils } from 'utils';
+import get from 'lodash/get';
+import capitalize from 'lodash/capitalize';
+
+const { request, flush, requests } = entityStoreActions;
+
+class FormUserClaims extends Component {
```

Comment:
In general, we want to avoid making new connected components when possible. I'm a little on the fence about this one. On the one hand, we could hoist these negotiations with the API up to People/Users/Edit, which is the connected component. I can see why you did it this way, in any case, because you want to share the connected logic between the two forms that are using this component. I'm not crazy about this solution, but I also don't have a better approach to offer, so for now let's leave it.